js/utils.js

```diff
@@ -1124,7 +1124,7 @@ let CombodoModal = {
 	 * @return {Object} The jQuery object representing the modal element
 	 * @api
 	 */
-	OpenUrlInModal: function(sTargetUrl, bCloseOtherModals, callbackOnContentLoaded) {
+	OpenUrlInModal: function(sTargetUrl, bCloseOtherModals, oCallbackOnContentLoaded) {
```

Comment:
```suggestion
	OpenUrlInModal: function(sTargetUrl, bCloseOtherModals, oCallbackOnContentLoaded) {
```

In that case, OCallbackOnContentLoaded isn't really an object, it's an anonymous function. We rather not prefix the type or define a prefix for this as a convention (but this has to be done with the whole team)